bin/docker-image-tool.sh

```diff
@@ -71,18 +71,29 @@ function build {
     --build-arg
     base_img=$(image_ref spark)
   )
-  local BASEDOCKERFILE=${BASEDOCKERFILE:-"$IMG_PATH/spark/Dockerfile"}
-  local PYDOCKERFILE=${PYDOCKERFILE:-"$IMG_PATH/spark/bindings/python/Dockerfile"}
-  local RDOCKERFILE=${RDOCKERFILE:-"$IMG_PATH/spark/bindings/R/Dockerfile"}
+  local BASEDOCKERFILE=${BASEDOCKERFILE:-"$IMG_PATH/main/dockerfiles/spark/Dockerfile"}
+  local PYDOCKERFILE=${PYDOCKERFILE:-"$IMG_PATH/main/dockerfiles/spark/bindings/python/Dockerfile"}
+  local KDOCKERFILE=${KDOCKERFILE:-"$IMG_PATH/test/dockerfiles/spark/kerberos/Dockerfile"}
```

Comment:
While, it's quite odd to add a Dockerfile here just for testing purpose.

Reply:
It is a docker image builder, so it seems to be an appropriate place. Especially, since it’s the first stage in our integration tests. But am open to opinions.

Reply:
Yes, but I don't think it's meant for also building images for tests. Can you create a separate script under `resource-managers/kubernetes/docker/src/test/scripts`?

Reply:
> Yes, but I don't think it's meant for also building images for tests.

I don't see why not. A simple flag would toggle it's build.,would that suffice? Having it be all in this script is cleaner given that the `image_ref` is defined in `docker-image-tool.sh`

Reply:
At the minimum, it might confuse people because people who check this script will possibly see there's now a `spark-keberos`  image without even noticing that the Dockerfile is meant for **tests**.  Given that we have the `resource-manager/kubernetes/integration-tests` directory, why not put test-specific Dockerfile and scripts under that directory?